### PR TITLE
Fix PageViewRegion Examples

### PR DESCRIPTION
**Pull request by dguy**
@miguno fixes for the PageViewRegion Examples

### CODE REVIEW

**Comment by miguno** [comment]
Can we also add instructions for writing input data with kafka avro console producer? And to read output via k-a-c-consumer?

**Comment by miguno** [comment]
Also, for Avro: where/how to access the schema files when running the cli commands because we are not assuming that schema registry is running to help with that. Alternatively we can include schema registry. But then we ahould also explain how to start and stop it. 

Perhaps we can point to the CP "main" quickstart for starting/stopping Kafka and ZK, and if we also want to leverage schema registry, then we can also point to the CP regy quickstart. Though I would only add these pointers as "for further details", and perhaps include the few commands to start broker, ZK, etc (in different terminals) directly in the examples. Idea: explain this centrally in kafka-streams/README , then refer to the README in the example's top javadocs. 

WDYT?

**Comment by miguno** [comment]
Ah sorry. I missed the driver which writes Avro data.

**Comment by miguno** [comment]
So other than the suggestion to also include broker/ZK instructions for start and stop this PR is +1 for me.

File: kafka-streams/src/main/java/io/confluent/examples/streams/PageViewRegionExample.java [hunk 1]
```diff
@@ -48,9 +50,21 @@
  * "PageViews" with a user profile table that reads from a topic named "UserProfiles" to compute the
  * number of page views per user region.
  *
- * Note: Before running this example you must 1) create the source topic (e.g. via
- * `kafka-topics --create ...`), then 2) start this example and 3) write some data to
- * the source topic (e.g. via `kafka-avro-console-producer`). Otherwise you won't see any data
+ * Note: Before running this example you must
+ * 1) Start Zookeeper, Kafka and Schema Registry
+ * 2) create the topics e.g.
+ * bin/kafka-topics --create --topic PageViews --zookeeper localhost:2181 --partitions 1
```
Comment (miguno) [inline kafka-streams/src/main/java/io/confluent/examples/streams/PageViewRegionExample.java]: We should add that these commands (ie without .sh extension) are for CP 3.0.0 only.  Apache Kafka scripts have the .sh extension.

File: kafka-streams/src/main/java/io/confluent/examples/streams/PageViewRegionExampleDriver.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Collections;
+import java.util.Properties;
+import java.util.Random;
+import java.util.stream.IntStream;
+
+import io.confluent.examples.streams.utils.WindowedStringDeserializer;
+
+public class PageViewRegionExampleDriver {
```
Comment (miguno) [inline kafka-streams/src/main/java/io/confluent/examples/streams/PageViewRegionExampleDriver.java]: We should also explain how to run this driver, eg when using the packaged jar (from mvn package).

Comment (miguno) [inline kafka-streams/src/main/java/io/confluent/examples/streams/PageViewRegionExampleDriver.java]: The explanation can be here or directly in the two PageView examples.

**Comment by dguy** [comment]
Made some further doco changes. I'm going to merge this in as is. We can always add more later.

**Pull request merged**